client/my-sites/earn/memberships/section.tsx

```diff
@@ -243,6 +253,7 @@ function MembershipsSection( { query }: MembershipsSectionProps ) {
 	return (
 		<div>
 			<QueryMembershipsSettings siteId={ site.ID } source={ source } />
+			<QueryMembershipsEarnings siteId={ site?.ID ?? 0 } />
```

Comment:
Without this, and if the user doesn't visit the Monetize home page first, the commissions above will be null because the relevant state isn't hydrated.